app/src/main/java/ch/epfl/sweng/favours/AuthentificationProcess.java

```diff
@@ -57,6 +57,11 @@ public void afterTextChanged(Editable s) {
             isEmailCorrect.set(isEmailValid(binding.emailTextField.getText().toString()));
```

Comment:
also when entering the E-Mail the first letter is capitalized which is not super bad as glitch but personally from the code in this PR I don't see why this happens. Maybe there is a field variable to turn this off.

Reply:
Don't see the issue. Tried to reproduce it and failed.